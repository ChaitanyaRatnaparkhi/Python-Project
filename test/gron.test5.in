{
  "company": {
    "name": "TechCorp",
    "departments": [
      {
        "name": "Engineering",
        "employees": [
          {"name": "Alice", "position": "Engineer"},
          {"name": "Bob", "position": "Manager"}
        ]
      },
      {
        "name": "Marketing",
        "employees": [
          {"name": "Charlie", "position": "Coordinator"}
        ]
      }
    ]
  }
}