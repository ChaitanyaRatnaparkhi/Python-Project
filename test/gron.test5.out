json = {}
json.company = {{}}
json.company.name = "TechCorp"
json.company.departments = []
json.company.departments[0] = {}
json.company.departments[0].name = "Engineering"
json.company.departments[0].employees = []
json.company.departments[0].employees[0] = {}
json.company.departments[0].employees[0].name = "Alice"
json.company.departments[0].employees[0].position = "Engineer"
json.company.departments[0].employees[1] = {}
json.company.departments[0].employees[1].name = "Bob"
json.company.departments[0].employees[1].position = "Manager"
json.company.departments[1] = {}
json.company.departments[1].name = "Marketing"
json.company.departments[1].employees = []
json.company.departments[1].employees[0] = {}
json.company.departments[1].employees[0].name = "Charlie"
json.company.departments[1].employees[0].position = "Coordinator"

